### DISCUSSION TITLE
can i use a the new stable diffusion models? like stable diffusion v2? if so how?

### DISCUSSION THREAD

**Discussion by Abrahm1234** [threestudio-project/threestudio#252] [Q&A]
can i use a the new stable diffusion models? like stable diffusion v2? if so how?

**Comment by bennyguo** [comment]
Sure! To use different SD models, you just need to change `system.prompt_processor.pretrained_model_name_or_path` and `system.guidance.pretrained_model_name_or_path` in the config. Actually, the current default configs are using SD2 by default.

**Comment by Abrahm1234** [comment]


**Reply by Abrahm1234** [reply]
linux mint 21, pip install torch==2.0.0+cu118 torchvision==0.15.1+cu118 torchaudio==2.0.1 --index-url https://download.pytorch.org/whl/cu118 , Driver Version: 520.61.05    CUDA Version: 11.8
